src/components/exercises/SyllableTable.js

```diff
@@ -8,56 +8,76 @@ import { RoundTextButton, RoundText } from '../Components';
 
 const mapIndexed = addIndex(map);
 
-const SyllableTable = ({ syllables, sound, index }) => {
+const SyllableTable = (
+  { letters, vowels, sound, indices: { letterIndex, vowelIndex } }
+) => {
+  let key = 0;
+
   const play = () => {
     sound.playAsync();
     sound.setPlaybackFinishedCallback(() => {
       sound.setPositionAsync(0);
     });
   };
 
-  return (
-    <View
-      style={{
-        flex: 1,
-        backgroundColor: PRIMARY,
-        alignItems: 'center'
-      }}
-    >
+  const createSyllables = (letter, letterKey) => {
+    return (
       <View
         style={{
           alignItems: 'center',
           justifyContent: 'center',
-          flexDirection: 'row',
-          flexWrap: 'wrap',
-          width: 300,
-          marginTop: 50
+          flexDirection: 'row'
         }}
+        key={letterKey}
       >
         {mapIndexed(
-          (syllable, key) => {
-            if (key === index) {
+          (vowel, vowelKey) => {
+            if (letterKey === letterIndex && vowelKey === vowelIndex) {
               return (
                 <RoundTextButton
                   style={{ margin: 5 }}
-                  text={syllable}
+                  text={letter + vowel}
```

Comment:
suggestion: I would move this out as `const syllable = letter + vowel` as it is used twice and captures the meaning nicely.